ads/_config.js

```diff
@@ -25,6 +25,7 @@ export const adPrefetch = {
   doubleclick: 'https://www.googletagservices.com/tag/js/gpt.js',
   a9: 'https://c.amazon-adsystem.com/aax2/assoc.js',
   adsense: 'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js',
+  mediaimpact: 'https://ec-ns.sascdn.com/diff/251/divscripte/t.js'
```

Comment:
At the moment it's for testing only, will be static to something like "https://ec-ns.sascdn.com/diff/251/amp/mediaimpact.js" when finished. Till then t.js does have everything we need for testing